src/pause.t.sol

```diff
@@ -204,67 +218,86 @@ contract Exec is Test {
 
     function testFail_delay_not_passed() public {
         address      usr = target;
+        bytes32      tag = codehash(usr);
         bytes memory fax = abi.encode(0);
         uint         eta = now + pause.delay();
 
-        pause.plot(usr, fax, eta);
-        pause.exec(usr, fax, eta);
+        pause.plot(usr, tag, fax, eta);
+        pause.exec(usr, tag, fax, eta);
     }
 
     function testFail_double_execution() public {
         address      usr = target;
+        bytes32      tag = codehash(usr);
         bytes memory fax = abi.encodeWithSignature("get()");
         uint         eta = now + pause.delay();
 
-        pause.plot(usr, fax, eta);
+        pause.plot(usr, tag, fax, eta);
         hevm.warp(eta);
-        pause.exec(usr, fax, eta);
-        pause.exec(usr, fax, eta);
+
+        pause.exec(usr, tag, fax, eta);
+        pause.exec(usr, tag, fax, eta);
     }
 
+    function testFail_tag_mismatch() public {
+        address      usr = target;
+        bytes32      tag = keccak256("hi");
```

Comment:
It took me a while to spot this (and therefore to understand why the test is supposed to be failing) as someone less familiar with the code - maybe it would be clearer if you removed the call to `keccak256`?

Reply:
updated.